### PR TITLE
Merge tag 'v221'

### LINKED ISSUE #67: Merge with upstream to upgrade from v205 to v221
https://github.com/heroku/heroku-buildpack-python/blob/main/CHANGELOG.md

### PR DESCRIPTION
**Pull request by EtienneM**
Sample deployment using this branch: https://dashboard.scalingo.com/apps/osc-fr1/biniou/deploy/537f3cdc-177b-4baf-8fb0-55c6d434ba91

Fix #67

### CODE REVIEW

File: bin/steps/python [hunk 1]
```diff
@@ -1,97 +1,114 @@
...
+    puts-warn "As such, it is no longer supported by the latest version of this buildpack:"
+    puts-warn "https://devcenter.heroku.com/changelog-items/2473"
+    puts-warn
+    puts-warn "You must either:"
+    puts-warn "  - Upgrade to Python 3 (recommended)"
+    puts-warn "  - Switch to the container stack and use the upstream legacy 'python:2.7' Docker images"
+    puts-warn "  - Switch to an older version of the Python buildpack (short term workaround only)"
+    puts-warn
+    puts-warn "For more details, see:"
+    puts-warn "https://devcenter.heroku.com/articles/python-2-7-eol-faq"
```
Comment (johnsudaar) [inline bin/steps/python:96]: Same here

File: bin/steps/python [hunk 2]
```diff
@@ -1,97 +1,114 @@
...
+    puts-warn "Python 2 reached upstream end-of-life on January 1st, 2020, and is"
+    puts-warn "therefore no longer receiving security updates. Apps still using it"
+    puts-warn "contain potential security vulnerabilities and should be upgraded to"
+    puts-warn "Python 3 as soon as possible."
+    puts-warn
+    puts-warn "In addition, Python 2 is only supported on our oldest stack, scalingo-18,"
+    puts-warn "which is deprecated and reaches end-of-life on April 30th, 2023."
+    puts-warn
+    puts-warn "As such, it is no longer supported by the latest version of this buildpack:"
+    puts-warn "https://devcenter.heroku.com/changelog-items/2473"
```
[the remark this answers is not in the document]
Comment (EtienneM) [reply bin/steps/python:88]: We don't have an equivalent to this page. Should I just replace with the link to the Python start doc?

[the remark this answers is not in the document]
Comment (johnsudaar) [reply bin/steps/python:88]: IMHO yes

**Review by johnsudaar — COMMENTED** [review]
Answered

**Review by johnsudaar — APPROVED** [review]

**Pull request merged**